Dalamud/Game/Network/MarketBoardUploaders/Universalis/UniversalisMarketBoardUploader.cs

```diff
@@ -59,7 +59,15 @@ internal class UniversalisMarketBoardUploader : IMarketBoardUploader {
                 }
 
                 var upload = JsonConvert.SerializeObject(listingsRequestObject);
-                client.UploadString(ApiBase + $"/upload/{ApiKey}", "POST", upload);
+                try
+                {
+                    client.UploadString(ApiBase + $"/upload/{ApiKey}", "POST", upload);
+                }
+                catch (WebException ex)
+                {
+                    int statusCode = (int)(ex.Response as HttpWebResponse).StatusCode;
+                    Log.Verbose($"Universalis returned an HTTP {statusCode}");
```

Comment:
These should be Log.Error and have ex attached, so you can still see the stack trace

Reply:
Changes made as requested.

Are the stacktraces really necessary in this instance? Outside of I guess the line number, the rest is pretty standard since its a non-200.